src/bootstrap/bootstrap.py

```diff
@@ -180,6 +180,7 @@ def parse_nightly_dates(self):
 
     def build_bootstrap(self):
         env = os.environ.copy()
+        env["CARGO_HOME"] = os.path.join(self.build_dir, "cargo")
```

Comment:
How come this was added? In theory this should pick up the same `CARGO_HOME` that it inherits?